etn-ccis/blui-react-native-workflows
# Feature/blui 5129 implement existing account success screen

Fixes 5129 .



#### Changes proposed in this Pull Request:

- [X] Write Test Case


- [X] Add JS Docs

- [X] Add Component in Example Project

- [X] Test component in dark theme/light theme 

- [X]  Test RTL View



#### Screenshots / Screen Recording (if applicable)

- 
<img width="440" alt="Screenshot 2024-02-27 at 6 34 49 PM" src="https://github.com/etn-ccis/blui-react-native-workflows/assets/133877691/5a2711e1-772c-4510-aef9-de905c48c6ad">

<img width="439" alt="Screenshot 2024-02-27 at 6 34 29 PM" src="https://github.com/etn-ccis/blui-react-native-workflows/assets/133877691/2001f9e8-acff-40d3-90a2-9026097d5a05">

<img width="443" alt="Screenshot 2024-02-27 at 6 34 00 PM" src="https://github.com/etn-ccis/blui-react-native-workflows/assets/133877691/e79db413-fa7b-4fe6-a4c0-3421598014c2">


#### To Test:

- Select Registration Provider menu
- Add organization name
- Click on next
- Check existingAccountSuccessScreen





#### Any specific feedback you are looking for?

-
